sound/soc/sof/intel/hda-loader.c

```diff
@@ -345,6 +398,12 @@ int hda_dsp_cl_boot_firmware(struct snd_sof_dev *sdev)
 	struct firmware stripped_firmware;
 	int ret, ret1, i;
 
+	if ((sdev->fw_ready.flags & SOF_IPC_INFO_D3_PERSISTENT) &&
+	    !sdev->first_boot) {
+		dev_info(sdev->dev, "info: IMR restore supported, booting from IMR directly\n");
```

Comment:
This will a very frequent and normal flow, so dev_info() is too verbose. Maybe dev_dbg() for this.